CUJExecutor/noProfileCuj

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
+
+#Copyright 2020 Google LLC
+#
+#Licensed under the Apache License, Version 2.0 (the "License");
+#you may not use this file except in compliance with the License.
+#You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+#Unless required by applicable law or agreed to in writing, software
+#distributed under the License is distributed on an "AS IS" BASIS,
+#WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#See the License for the specific language governing permissions and
+#limitations under the License.
+
+
+INPUT_FILE=${1:-input.txt}
+FILE_NAME="${INPUT_FILE%.*}"
+PRE_CUJ=$(sed '1q;d' $INPUT_FILE)
+RECORD=$(sed '4q;d' $INPUT_FILE)
+
+echo "Resetting the profile...."
+adb shell cmd package compile --reset com.google.android.apps.maps
+
+#Get the preparatory action into the profile:
+echo "Walking through the CUJ to (hopefully) add it to the profile...."
+echo $PRE_CUJ > walk_through_precuj.txt
+echo "" >> walk_through_precuj.txt
+echo "0" >> walk_through_precuj.txt
+./executeCuj walk_through_precuj.txt
+
+
+echo "Waiting..."
+sleep 60
```

Comment:
Should force-stop the app before compiling.  Not clear if `executeCuj` will be doing that.

Reply:
does the force-stop need to come before compiling? or can it come after? (executeCuj will always force stop an app before opening it)

Reply:
Should stop before compiling, to put the app in a known state.